ministryofjustice/analytics-platform-ops
# Add IAM role support for kubernetes pods

## What

- Adds [kube2iam](https://github.com/jtblin/kube2iam) daemonset Helm chart

## How to review

1. Install kube2iam Helm chart in environment
2. Create test IAM role in AWS with permissions to list an existing S3 bucket
3. Start [test kubernetes pod](https://github.com/jtblin/kube2iam#kubernetes-annotation) with `iam.amazonaws.com/role: role-name` annotation, with edited s3 command to list bucket
3. Check pod logs for S3 bucket listing

Provide [http://example.com](links) to relevant tickets, articles or other
resources.

Note the Trello Github integration links tickets to PRs/branches/commits (in
both directions).
